src/App.jsx

```diff
@@ -14,45 +16,89 @@ export const goodsFromServer = [
   'Garlic',
 ];
 
-export const App = () => (
-  <div className="section content">
-    <div className="buttons">
-      <button
-        type="button"
-        className="button is-info is-light"
-      >
-        Sort alphabetically
-      </button>
-
-      <button
-        type="button"
-        className="button is-success is-light"
-      >
-        Sort by length
-      </button>
-
-      <button
-        type="button"
-        className="button is-warning is-light"
-      >
-        Reverse
-      </button>
-
-      <button
-        type="button"
-        className="button is-danger is-light"
-      >
-        Reset
-      </button>
-    </div>
+const SORT_ALPHABETICALLY = 'sort alphabetically';
+const SORT_BY_LENGTH = 'sort by length';
+
+const sortByCondition = (condition, isReversed) => {
+  const copyGoods = [...goodsFromServer];
```

Comment:
IMHO goodsCopy seems more like array constant.
It is good, but it looks a bit like function name. 
Explanation: `Do what? copy goods` `What is it? goods copy`